### PR TITLE
rust: release v0.3.3

### PR DESCRIPTION
**Pull request by abrown**
Bump the version of the Rust crates to release FreeBSD support.

### CODE REVIEW

File: rust/ittapi-sys/README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ Profiling API. More details about `ittapi` are available on its [README].
 [README]: https://github.com/intel/ittapi#readme
 
 > IMPORTANT NOTE: this crate is currently only tested on Linux, macOS, and Windows platforms but
-> support for other platforms is intended; contributions are welcome!
+> support for other platforms is intended; contributions are welcome! FreeBSD is supported but
+> untested.
```
Comment (jlb6740) [inline rust/ittapi-sys/README.md:18]: Not sure if you intended for this newline but I guess it doesn't matter in markup.

File: rust/ittapi/README.md [hunk 2]
```diff
@@ -21,7 +21,8 @@ This uses the [`ittapi-sys`] crate which depends on the [C `ittapi` library].
 [C `ittapi` library]: https://github.com/intel/ittapi
 
 > IMPORTANT NOTE: this crate is currently only tested on Linux, macOS, and Windows platforms but
-> support for other platforms is intended; contributions are welcome!
+> support for other platforms is intended; contributions are welcome! FreeBSD is supported but
+> untested.
```
Comment (jlb6740) [inline rust/ittapi/README.md:25]: Same comment as above

**Review by jlb6740 — APPROVED** [review]

File: rust/ittapi-sys/README.md (hunk 1 above)
Comment (abrown) [reply rust/ittapi-sys/README.md:18]: Yeah, it's fine.

**Comment by lin72h** [comment]
Thank for making it work on FreeBSD, Is there any port for this

**Comment by abrown** [comment]
@lin72h, not sure I understand: "any port"?

**Comment by lin72h** [comment]
@abrown sorry for the confusion, by "Port", I mean Freebsd's [Port Collections](https://en.wikipedia.org/wiki/FreeBSD_Ports), which build and install software easier

**Comment by abrown** [comment]
Oh, I see! Yeah, I do not have much knowledge about the FreeBSD ecosystem but I would guess there is no port for these Rust bindings. If you are interested in making one, you would want it to contain the top-level C project, not these Rust bindings. The best way to get feedback on whether all of the maintainers want to do something like that would be to open an issue for it! (But guessing that, like me, they are not FreeBSD users I would probably recommend just creating the port yourself).

**Pull request merged**